eta-meeting-organizer-frontend/src/app/login/components/login.component.ts

```diff
@@ -73,42 +49,23 @@ export function provideConfig() {
           <mat-card-title>{{ "login.title" | translate }}</mat-card-title>
           <br />
           <mat-card-content>
+            <a href="http://localhost:8080/oauth2/authorize/google">
```

Comment:
be van égetve az url, szivja fel a konfigból

Reply:
Javítva